crt_portal/cts_forms/views.py

```diff
@@ -48,8 +48,6 @@ def error_400(request, exception=None):
         request,
         'forms/errors.html', {
             'status': 400,
-            'message': _("Bad request"),
-            'helptext': _("It seems your browser is not responding properly. Try refreshing this page.")
```

Comment:
It strikes me that these messages and helptext are worth preserving and that we can keep them somewhere in the template. Are we deliberately moving away from specific error messages and being more generic? I'm not sure I agree with this since users will have little frame of reference other than literally, "something went wrong". How will users communicate specifics with us if an error does, in fact, arise?